sources/packages/backend/apps/workers/src/controllers/supporting-user/supporting-user.controller.ts

```diff
@@ -132,7 +147,55 @@ export class SupportingUserController {
       >
     >,
   ): Promise<MustReturnJobActionAcknowledgement> {
-    return job.fail("To be implemented.");
+    const jobLogger = new Logger(job.type);
+    try {
+      const application = await this.applicationService.getApplicationById(
+        job.variables.applicationId,
+        { loadDynamicData: true },
+      );
+      if (!application) {
+        const message = "Application ID not found.";
+        jobLogger.error(message);
+        return job.error(APPLICATION_NOT_FOUND, message);
+      }
+      // Resolve the full name from the application dynamic data using the optional provided filter.
+      let fullName: string;
+      if (job.variables.fullNamePropertyFilter) {
+        [fullName] = JSONPath.query(
```

Comment:
What about `JSONPath.value` ?

Reply:
Tested and it works in the same way. I will change it.